deno/send_message/src/functions/send_message_discord_webhook.ts

```diff
@@ -0,0 +1,23 @@
+export default async function (env: any, message: string) {
+  const webhook = env["DISCORD_WEBHOOK_URL"];
```

Comment:
Would you please update `env` to `variables`? and the type can probably be more specific than `any`. Same with the other functions please.

Reply:
ok, I will change these tomorrow morning!!
It is already late